README.org

```diff
@@ -18,7 +18,7 @@ Gaius is *not* stable or well tested. Only use it if you're
 adventurous.
 
 
-Note: Gaius.jl is not actively maintained and I do not anticipate doing further work on it. There are other, more promising projects that may result in a scalable, multi-threaded pure julia BLAS library such as [[https://github.com/mcabbott/Tullio.jl][Tullio.jl]].
+Note: Gaius.jl is not actively maintained and I do not anticipate doing further work on it. There are other, more promising projects that may result in a scalable, multi-threaded pure julia BLAS library such as [[https://github.com/mcabbott/Tullio.jl][Tullio.jl]] and [[https://github.com/chriselrod/PaddedMatrices.jl][PaddedMatrices.jl]].
```

Comment:
```suggestion
Note: Gaius.jl is not actively maintained and I do not anticipate doing further work on it. There are other, more promising projects that may result in a scalable, multi-threaded pure julia BLAS library such as [[https://github.com/mcabbott/Tullio.jl][Tullio.jl]] and [[https://github.com/chriselrod/PaddedMatrices.jl][PaddedMatrices.jl]]. 

However, you may find this library useful as relatively simple playground for learning about the implementation of linear algebra routines.
```

Reply:
I think maybe a note like this would be good, but yeah the message you added is great.